sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -226,35 +254,18 @@ def __init__(  # pylint: disable=missing-client-constructor-parameter-credential
             is "2019-02-02".
         :paramtype api_version: str or None
         """
-        super(TablesBaseClient, self).__init__(endpoint, credential=credential, **kwargs)  # type: ignore
-        self._client = AzureTable(self.url, policies=kwargs.pop("policies", self._policies), **kwargs)
-        self._client._config.version = get_api_version(kwargs, self._client._config.version)  # type: ignore # pylint: disable=protected-access
+        super(TablesBaseClient, self).__init__(endpoint, credential=credential, **kwargs)
+        self._client = AzureTable(
+            self.url, policies=kwargs.pop("policies", self._policies), version=self._api_version, **kwargs
```

Comment:
Are we sure passing in `version` here is working the same way?
I think we originally had to use line 231 above for a reason - but maybe that reason no longer applies after the code regen?

Reply:
Reverted